### PR TITLE
Fully-qualified path to 'test' executable

### PR DESCRIPTION
**Pull request by drewhemm**

### CODE REVIEW

File: manifests/update.pp [hunk 1]
```diff
@@ -2,7 +2,7 @@
   exec { 'supervisor::update':
     command     => '/usr/bin/supervisorctl update',
     logoutput   => on_failure,
-    onlyif      => "test -f ${supervisor::unix_server_file}",
+    onlyif      => "/usr/bin/test -f ${supervisor::unix_server_file}",
```
Comment (plathrop) [inline manifests/update.pp]: This would probably be better as:
```puppet
onlyif => "/usr/bin/env test -f ${supervisor::unix_server_file}",
```

**Comment by plathrop** [comment]
:+1: could be improved, but happy with it as-is too. Let me know if you want me to merge as-is or if you want to update it.

**Comment by drewhemm** [comment]
I've made the change you suggested and it works fine!

**Comment by plathrop** [comment]
Awesome.

Merging.

**Pull request merged**